service.go

```diff
@@ -297,3 +301,22 @@ func (s *serviceManager) serviceByID(id ServiceID) (Service, bool) {
 	}
 	return serv, true
 }
+
+// NewProtocol contains the logic of how and where a ProtocolInstance is
+// created. If the token's ServiceID is nil, then onet handles the creation of
```

Comment:
I propose to write "ONet" everywhere, what do you think?
Again, please don't use abbreviations in the comment for PI and other elements.